apa-annotated-bibliography.csl

```diff
@@ -51,11 +51,21 @@
   <locale xml:lang="de">
     <terms>
       <term name="et-al">et al.</term>
+      <term name="letter">persönliche Kommunikation</term>
+      <term name="letter" form="short">Brief</term>
     </terms>
   </locale>
   <locale xml:lang="es">
     <terms>
       <term name="from">de</term>
+      <term name="letter">persönliche Kommunikation</term>
+      <term name="letter" form="short">carta</term>
+    </terms>
+  </locale>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="letter">comunicación personal</term>
```

Comment:
(nor is this; I think that should be "communication personnelle" but as I've mentioned, my French is pretty basic.

Reply:
Yeah, copy-pasted into the french field instead of spanish..